gapic/templates/%namespace/%name_%version/%sub/services/%service/async_client.py.j2

```diff
@@ -243,6 +251,24 @@ class {{ service.async_client_name }}:
 
         )
 
+        if CLIENT_LOGGING_SUPPORTED and _LOGGER.isEnabledFor(std_logging.DEBUG):  # pragma: NO COVER
+            # TODO: Make this performant when logging is not enabled
```

Comment:
```suggestion
```

Remove this TODO comment as it was from before we added the guard for `_LOGGER.isEnabledFor(std_logging.DEBUG)`